Tests/Functional/Command/CreateTestDataCommandTest.php

```diff
@@ -41,6 +41,8 @@ protected function setUp(): void
     #[Test]
     public function isConsoleCommand(): void
     {
+        // The CreateTestDataCommand extends Command, so this test is always true
+        // This test validates the inheritance structure is correct
```

Comment:
I don't think we should have a command above each and every `assertInstanceOf` in the tests. The fact that we have that call implicitly communicates that we want to check the inheritance chain.

Ditto for the other added commands above the other `assertInstanceOf` calls.